src/components/cardbuilder/cardBuilder.js

```diff
@@ -1135,49 +1241,37 @@ define(['datetime', 'imageLoader', 'connectionManager', 'itemHelper', 'focusMana
             return counts.join(', ');
         }
 
-        function getProgramIndicators(item) {
-
-            item = item.ProgramInfo || item;
-
-            var html = '';
+        let refreshIndicatorLoaded;
```

Comment:
This refresh indicator is terrible in my opinion. We should also think about treating it as a task on the server so refresh actions will show up on the dashboard as well rather than be completely hidden most of the time.

Reply:
Agreed, but outside the scope of this :)